.organizationmap

```diff
@@ -333,6 +333,7 @@ Pacific Northwest National Laboratory <contact@pnl.gov> Evan Felix <evan.felix@p
 Pacific Northwest National Laboratory <contact@pnl.gov> Scott Devoid <devoid@anl.gov>
 Piston Cloud Computing <info@pistoncloud.com> Mike Lundy <mike@fluffypenguin.org>
 Pogoapp <contact@pogoapp.com> Paul Meserve <paul@pogodan.com>
+Proxmox Server Solutions GmbH <contcat@proxmox.com> Fabian Grünbichler <f.gruenbichler@proxmox.com>
```

Comment:
There is a typo here ("contcat" vs "contact"), but if possible, "office@proxmox.com" would be the more appropriate organizational address.

Thanks!